drivers/hmis/spec/factories/hmis/hud/enrollments.rb

```diff
@@ -31,7 +31,16 @@
       exit_date { nil }
     end
     after(:create) do |enrollment, evaluator|
-      enrollment.exit = create(:hmis_hud_exit, exit_date: evaluator.exit_date, enrollment: enrollment, data_source: enrollment.data_source, client: enrollment.client) if evaluator.exit_date
+      if evaluator.exit_date
+        enrollment.exit = create(
+          :hmis_base_hud_exit,
```

Comment:
A few changes here:
- changed the `enrollment` factory to use `hmis_base_hud_exit` instead of `hmis_hud_exit`, which does some manipulation with the exit date provided. I *think* I tracked down the other usages of this exit date callback, and they are not impacted negatively.
  - client_active_enrollment_spec.rb
  - bulk_assign_service_spec.rb
  - open_enrollment_summary_spec.rb
- also added the `without_validations` trait to the creation call. If you think it's worth the effort, maybe it would be good to make this depend on a `without_validations` trait passed in to the `enrollment` factory?

Reply:
Good catch, thanks! I think it's fine to skip validation here because the caller is explicitly passing an exit date.

Reply:
We can use `skip_validate`, though, see below